behave/features/steps/specialist_provider.py

```diff
@@ -366,5 +370,64 @@ def step_impl(context):
     assert CASE_SPLIT_TEXT in full_case_log, "Could not find split case in operators dashboard"
 
 
+@step(u'I am on the CSV upload page')
+def step_impl(context):
+    csv_page = f"{CLA_FRONTEND_URL}{CLA_FRONTEND_CSV_URL}"
+    context.helperfunc.open(csv_page)
+    page = CLA_FRONTEND_CSV_URL
+    wait_until_page_is_loaded(page, context)
+    header = context.helperfunc.find_by_xpath("//header/h2[text()='CSV Upload']")
+    assert header is not None
+
+
+@step(u'I select \'Choose file\' and upload a csv file')
+def step_impl(context):
+    input_csv = context.helperfunc.find_by_xpath("//input[@name='csvfile']")
+    # test will fail if file can't be found or upload fails.
+    input_csv.send_keys(os.getcwd() + CLA_SPECIALIST_CSV_UPLOAD_PATH)
+
+
+@step(u'I select the month and year for the uploaded csv file')
+def step_impl(context):
+    # use setter method to set up the datetime in the Helperfunc class
+    context.helperfunc.set_date_now(datetime.datetime.now().date())
```

Comment:
Declare the date you want (as in date at the start of the month) in the __init__ function of helperfunc
`self.date_start_this_month = datetime.datetime.now().date().replace(day=1, month=date_now.month - 1)`

Reply:
Ah okay, I'm doing it the Java way again here. Changed.